overrides/default.nix

```diff
@@ -2141,6 +2141,34 @@ lib.composeManyExtensions [
         buildInputs = (old.buildInputs or [ ]) ++ [ pkgs.taglib ];
       });
 
+      pytesseract =
+        let
+          pytesseract-cmd-patch = pkgs.writeText "pytesseract-cmd.patch" ''
+            --- a/pytesseract/pytesseract.py
+            +++ b/pytesseract/pytesseract.py
+            @@ -27,7 +27,7 @@
+             from PIL import Image
+
+
+            -tesseract_cmd = 'tesseract'
+            +tesseract_cmd = '${pkgs.tesseract4}/bin/tesseract'
+
+             numpy_installed = find_loader('numpy') is not None
+             if numpy_installed:
+          '';
+        in
+        super.pytesseract.overridePythonAttrs (old: {
+          buildInputs = (old.buildInputs or [ ]) ++ [ pkgs.tesseract4 ];
```

Comment:
Does this not need to be a `propagatedBuildInput`, to be available at runtime? In that case you might not need the patch.

Reply:
Good question, let me try it. I saw that the [pytesseract default.nix in Nixpkgs][1] used `buildInputs`, but I don't know if that's the best way to do it.

For reproducibility it seems better to have an explicit path to the executable rather than relying on the `PATH` variable where the script runs, so maybe that's the consideration.

[1]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/pytesseract/default.nix

Reply:
Okay, I tried changing it to `propagatedBuildInputs`  and removing the patch, and the test case I wrote failed:

```
pytesseract.pytesseract.TesseractNotFoundError: tesseract is not installed or it's not in your PATH. See README file for more information.
```